source/agora/utils/SCPPrettyPrinter.d

```diff
@@ -317,30 +301,12 @@ private struct SCPStatementFmt
 /// Formatting struct for `SCPEnvelope`
 private struct SCPEnvelopeFmt
 {
-    /// Pointer to SCPEnvelope
-    private const SCPEnvelope* envelope;
```

Comment:
The reason why this was a pointer was to avoid memory copies.

This is even more important now that the signature is a static array.

Reply:
@Geod24, @AndrejMitrovic  Okay. I'll remove the last commit.